DICOMcloud/DICOMcloud/JsonDicomConverter.cs

```diff
@@ -483,13 +492,32 @@ int level
                         TransferSyntaxUID = values.FirstOrDefault ( ) ; 
                     }
 
-                    dataset.AddOrUpdate<string> ( vr, tag, values.ToArray ( ) );                
+                    dataset.AddOrUpdate<string> ( vr, tag, System.Text.Encoding.Default ,values.ToArray ( ) );                
                 }
                 break ;
             }
 
         }
 
+
+        protected virtual void ReadVr_DS(JsonTextReader reader, DicomTag tag, DicomDataset dataset)
+        {
+
+            List<string> values = new List<string>();
+
+            while (reader.Read() && reader.TokenType == JsonToken.StartArray)
```

Comment:
Can you please explain more what is the problem you are trying to solve here and why the DS VR needs a special handling?
Anyway we can test this update?

Reply:
Those values are affected by local culture settings, so in my case it would strip the point from a decimal completly or in the other case exchange it for a comma. So for the decimal case the value can be directly added to the dataset after this. On the other hand for the default string case in ReadDefaultVr I added  System.Text.Encoding.Default to dataset.add this should handle culture specific characters but not the conversion of decimals.